lib/ProductOpener/Tags.pm

```diff
@@ -718,6 +719,34 @@ sub get_tags_grouped_by_property ($tagtype, $tagids_ref, $prop_name, $props_ref,
 	return $grouped_tags;
 }
 
+=head2 get_all_tags_having_property ($product_ref, $tagtype, $prop_name)
+
+For each tag of a given field ($tagtype, can be "labels" or "categories", for example),
+and a given property ($prop_name, without last column (:). Can be "incompatible_with:en", for example),
+return a hash of tagid <-> property_value
```

Comment:
I don't know if it's what's wanted, but it does not handle property inheritance. 

```suggestion
return a hash of tagid <-> property_value

Note that this does not handle inheritance.
```